### PR TITLE
Fixes #240 Update heroku deployment with mlab

### LINKED ISSUE #240: Update heroku deployment with changes in app.json
### This is a
- [ ] Bug report
- [ ] Feature request

### Description

The app.json file needs to be updated with the features added to yaydoc in the PR https://github.com/fossasia/yaydoc/pull/219

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue #240 
## Motivation and Context

Without this change it would've become impossible to perform a One-Click deployment from the repository's readme
## How Has This Been Tested?

The app https://yaydoc7.herokuapp.com was deployed using the Button
## Types of changes

- [ ] Bug fix 
- [ ] New feature
- [x] Enhancement
## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by pri22296** [comment]
@imujjwal96 404 on clicking on CI deploy

**Comment by imujjwal96** [comment]
@pri22296 That is because of a random github id's i added. You may check at the main site. I deployed my local changes their as well.

File: app.json [hunk 1]
```diff
@@ -65,5 +68,8 @@
     {
       "url": "https://github.com/heroku/heroku-buildpack-cli"
     }
+  ],
+  "addons": [
+    "mongolab"
```
Comment (sch00lb0y) [inline app.json]: I think thIs addon will add `MONGO_URI` variable directly to the environment. I'm not sure
- if it does there is no point of asking separately in deployment form
- if not what is the use of this addon
Please clarify my doubt

Comment (imujjwal96) [inline app.json]: Oh! didn't notice that! Thanks!!!!

Comment (imujjwal96) [inline app.json]: Updated!

**Pull request merged**